SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialDispatcher.cpp

```diff
@@ -44,30 +44,24 @@ void SpatialDispatcher::ProcessOps(const TArray<Worker_Op>& Ops)
 		{
 		// Critical Section
 		case WORKER_OP_TYPE_CRITICAL_SECTION:
-			Receiver->OnCriticalSection(Op.op.critical_section.in_critical_section != 0);
 			break;
 
 		// Entity Lifetime
 		case WORKER_OP_TYPE_ADD_ENTITY:
-			Receiver->OnAddEntity(Op.op.add_entity);
 			break;
 		case WORKER_OP_TYPE_REMOVE_ENTITY:
-			Receiver->OnRemoveEntity(Op.op.remove_entity);
 			StaticComponentView->OnRemoveEntity(Op.op.remove_entity.entity_id);
-			Receiver->DropQueuedRemoveComponentOpsForEntity(Op.op.remove_entity.entity_id);
 			break;
 
 		// Components
 		case WORKER_OP_TYPE_ADD_COMPONENT:
 			StaticComponentView->OnAddComponent(Op.op.add_component);
-			Receiver->OnAddComponent(Op.op.add_component);
 			break;
 		case WORKER_OP_TYPE_REMOVE_COMPONENT:
-			Receiver->OnRemoveComponent(Op.op.remove_component);
```

Comment:
Did we previously never remove individual components from the scv??

Reply:
Oh, it's done in the receiver

Reply:
aha yeah it used to be in the receiver